samples/getting-started/intro-to-iqsharp/README.md

```diff
@@ -0,0 +1,28 @@
+---
+page_type: sample
+languages: [qsharp]
+products: [qdk]
+---
+
+# Intro to IQ# #
```

Comment:
I would suggest renaming this to something like: "Intro to Q# Jupyter notebooks"

Reply:
Good point, will go on and rename, thanks!